PhysicsTools/PatAlgos/plugins/PATSecondaryVertexSlimmer.cc

```diff
@@ -15,32 +15,32 @@
 #include "DataFormats/Common/interface/RefToPtr.h"
 
 namespace pat {
-    class PATSecondaryVertexSlimmer : public edm::EDProducer {
-        public:
-            explicit PATSecondaryVertexSlimmer(const edm::ParameterSet&);
-            ~PATSecondaryVertexSlimmer();
-
-            virtual void produce(edm::Event&, const edm::EventSetup&);
-        private:
-            edm::EDGetTokenT<reco::VertexCompositePtrCandidateCollection> src_;
-            edm::EDGetTokenT<std::vector<reco::Vertex> > srcLegacy_;
-            edm::EDGetTokenT<edm::Association<pat::PackedCandidateCollection> > map_;
-            edm::EDGetTokenT<edm::Association<pat::PackedCandidateCollection> > map2_;
+  class PATSecondaryVertexSlimmer : public edm::global::EDProducer<> {
```

Comment:
Other than a ```std::cout``` call in produce which needs to be changed to the message logger (```std::cout``` isn't thread safe) it looks OK.